pkg/api/v1/types.go

```diff
@@ -1965,11 +1965,15 @@ type KubevirtNodeSpec struct {
 	// SecondaryDisks contains list of secondary-disks
 	SecondaryDisks []SecondaryDisks `json:"secondaryDisks"`
 	// PodAffinityPreset describes pod affinity scheduling rules
+	// Deprecated
 	PodAffinityPreset string `json:"podAffinityPreset"`
 	// PodAntiAffinityPreset describes pod anti-affinity scheduling rules
+	// Deprecated
 	PodAntiAffinityPreset string `json:"podAntiAffinityPreset"`
```

Comment:
Please add a new line above and add a reason for depreaction, something like
> // PodAffinityPreset describes pod affinity scheduling rules
   //
   // Deprecated: in favor of topology spread constraints